upup/models/cloudup/resources/addons/storage-aws.addons.k8s.io/v1.15.0.yaml

```diff
@@ -14,13 +14,28 @@ apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
   name: gp2
+  labels:
+    k8s-addon: storage-aws.addons.k8s.io
+provisioner: kubernetes.io/aws-ebs
+parameters:
+  type: gp2
+
+---
+
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: gp2-kops
```

Comment:
Sorry to nit-pick... I'm wondering if we should use a consistent prefix, rather than a suffix.  We've started doing this with e.g. clusterroles, and there we've used `kops:` but I don't know if the colon is available everywhere (?).  WDYT about `kops-gp2` or `kops:gp2` in general?

Reply:
I think that is a good idea to start with something consistent and move forward with it. Unfortunately it won't let us do `kops:gp2` but I think `kops-gp2` make sense @justinsb 

Thoughts @hakman

Reply:
`kops-gp2-1-18`? I am sure we will change it again someday. :)

Reply:
I do think it's OK to change these going forwards; "kops-gp2" can mean "the gp2 class, but otherwise whatever kops recommends".   We should probably have a note that if people want to lock in a specific behaviour, they should probably use their own storage class (though to be honest I'm not sure how users could do that without knowing what features are coming in the future).

This is sort of philosophy with most of the kops fields - if you leave it unspecified, that means "whatever kops recommends" (I sometimes call it omakase).  But if you care, you should specify it, in which case kops honors your intention.  We can - and do - change the defaults, but if you specify a field we either honor it or it is an error.

Reply:
(BTW, if we wanted to, we could also name this `kops-ssd`, and then offer storage classes that work across clouds, though I personally find it better to just use the default storage class in my apps)

Reply:
> (BTW, if we wanted to, we could also name this `kops-ssd`, and then offer storage classes that work across clouds, though I personally find it better to just use the default storage class in my apps)

I do like the idea of `kops-ssd` as a starting point. Should I add a version to it as well? I would like to see this go out whenever it can, so should I target `1.16` or `1.17`? Therefore the name could be:

`kops-ssd-1-16` for instance

Reply:
I would prefer we target 1.17. This is too featureish to go into 1.16 at this time.

Reply:
Done @johngmyers @justinsb